Cue/app/tabs/library/LibraryHome.js

```diff
@@ -44,20 +44,24 @@ const styles = {
 }
 
 type Props = {
-  decks: Array<Deck>,
+  decks: Array<Deck>,
   navigator: Navigator,
   onPressMenu?: () => void,
-  localChanges: {},
-
-  // From Redux:
+
+  // From Redux:
+  localChanges: {},
+  inaccessibleDecks: ?Array<Deck>,
+
   onCreateDeck: (string) => any,
   onLoadLibrary: () => any,
   onSyncLibrary: (any) => any,
-  deleteDeck: (uuid: string) => any,
+  deleteDeck: (uuid: string) => any,
+  onClearInaccessibleDecks: () => any,
+  onDeckCopy: (deck: Deck) => any,

```

Comment:
`onCopyDeck`?

Also, feel free to change `deleteDeck` to `onDeleteDeck` to be consistent.